internal/pkg/manifest/lb_fargate_manifest.go

```diff
@@ -56,6 +56,7 @@ type LogsConfig struct {
 // RoutingRule holds the path to route requests to the service.
 type RoutingRule struct {
 	Path string `yaml:"path"`
+	HealthCheckPath string `yaml:"healthcheck"`
```

Comment:
Maybe we should use underscores like `health_check` - what do you think?

Reply:
NO